# TP 2 - Jupyter 🪐

| Champ | Information |
|---|---|
| **Nom** | Benamira Ahmed Ouassim |
| **Numéro étudiant** | 300150564 |
| **Cours** | INF1093-201-26H-05 |

In [1]:
import sys

print("=== Informations étudiant ===")
print("Nom        : Benamira Ahmed Ouassim")
print("Numéro     : 300150564")
print("Cours      : INF1093-201-26H-05")
print()
print("=== Environnement ===")
print(f"Python     : {sys.version}")
print(f"Exécutable : {sys.executable}")

=== Informations étudiant ===
Nom        : Benamira Ahmed Ouassim
Numéro     : 300150564
Cours      : INF1093-201-26H-05

=== Environnement ===
Python     : 3.12.9 | packaged by conda-forge | (main, Mar  4 2025, 22:37:18) [MSC v.1943 64 bit (AMD64)]
Exécutable : C:\tools\miniforge3\python.exe


## ✅ Résumé de l'installation

| Étape | Statut |
|---|---|
| Chocolatey installé | ✅ |
| Miniforge installé via Chocolatey | ✅ |
| PATH configuré dans `$PROFILE` | ✅ |
| `conda init powershell` configuré | ✅ |
| Environnement `INF1093` créé (Python 3.12) | ✅ |
| JupyterLab installé via conda-forge | ✅ |